### ISSUE TITLE
Localized fields nested in SuperTable blocks overwritten by source values when publishing multiple languages

### ISSUE THREAD

**Issue by sidedwards** [engram-design/SuperTable#434]
## 🐛 Bug description

As the title describes, localized fields nested within SuperTable blocks are getting overwritten by source values when publishing drafts in multiple languages.

## 👣 Reproduction steps

1. Create a SuperTable block type.
2. Add localized fields within the SuperTable block.
3. Open an Entry and create two separate drafts.
4. In the primary site of the first draft, update the localized field values within the SuperTable block.
5. In the second draft, switch to another site and update the localized field values within the SuperTable block.
6. Apply the first draft.
7. Localized field values should reflect updates from step 4.
8. Apply the second draft.
9. See error. The localized field value updates from the first draft are reverted to the original content.

**Why does this happen?**

I believe this is related to https://github.com/verbb/super-table/issues/432 and the recent updates in Craft 3.7+. Since blocks received new IDs on each save/draft apply, localized field content that is stored within a particular block ID is lost when a new draft is applied.

### 📚 Resources

**Additional info**
- Plugin version: 2.6.8
- Craft version: 3.7.14
- Multi-site: Yes

https://user-images.githubusercontent.com/11489608/137371508-a02b50c3-5b2f-428c-97db-a7d66d74468a.mp4

### 🔗 References
- https://github.com/craftcms/cms/pull/7710
- https://github.com/verbb/super-table/issues/432

**Comment by engram-design** [comment]
I'm been porting over a number of content changes caused by Craft 3.7+ changes, and would be great if you're able to test these out. To get the fixes early, change your `verbb/super-table` requirement in `composer.json` to:

```
"require": {
  "verbb/super-table": "dev-craft-3 as 2.6.8",
  "...": "..."
}
```

Then run `composer update`.

**Comment by sidedwards** [comment]
@engram-design, thanks for the update; let me test this out locally.

**Comment by alanski66** [comment]
@engram-design I have tested this on an icon saving field (Dolphiq jcon picker plugin). The field would select but not save the icon selected in the supertable field. They appear to work in draft but if I click apply draft then the icons selected are lost

**Comment by engram-design** [comment]
Should be fixed in [2.7.0](https://github.com/verbb/super-table/releases/tag/2.7.0)
